deploy/helm/sumologic/values.yaml

```diff
@@ -1697,13 +1697,7 @@ kube-prometheus-stack:
     ## Use the GCR repo, it's more recent and has ARM images starting from 1.9.8
     image:
       repository: k8s.gcr.io/kube-state-metrics/kube-state-metrics
-      tag: v1.9.8
-    # Do not collect metrics for CertificateSigningRequest or Ingress resources.
-    # This is because the v1 version of kube-state-metrics uses API versions
-    # that are removed in Kubernetes v1.22.
-    collectors:
-      certificatesigningrequests: false
-      ingresses: false
+      tag: v2.3.0
```

Comment:
Do we need to specify the `repository` ang `tag`? Maybe we could just use the defaults now?

Reply:
Also I believe we could remove the `kube-prometheus-stack.prometheus-node-exporter.image.tag`?